fundamentals/computing-development-environments/index.md

```diff
@@ -0,0 +1,3 @@
+# Computing and Development Environments
+
+Coming soon
```

Comment:
```suggestion
Coming soon

Table of contents

```{tableofcontents}
```

```

Let's add table of contents here. See https://jupyterbook.org/en/stable/structure/configure.html#add-a-table-of-contents-to-a-pages-content

Reply:
Edited your comment for formatting

did you know that if you needed a code block with 3 backticks inside of it to display properly, you could make the code block with 4 backticks and it'll work? i just learned today, wild!